set-quotas.py

```diff
@@ -51,8 +51,22 @@ def parse_rows(rows):
     # NOTE: entry[17] is a required field in the Google Form, so it is safe
     # to assume entry[0:16] exists.
 
+    PROJECT_COLUMN = 7
     project_list = []
-    for idx, entry in enumerate(rows):
+
+    if select_project:
+        try:
+            rows = select_rows(select_project, PROJECT_COLUMN, rows)
+            if len(rows) > 2:
+                print ("WARNING: Multiple requests found for project {}. All "
+                       "{} requests will be processed. You may need to close "
+                       "multiple tickets.").format(args.project, len(rows))
```

Comment:
Oh wait, I see what you were pointing out - I missed that the -1 was missing (which is weird because I'm pretty sure I copy-pasted...) 

I'll fix it in some future PR.